stories/components/Card.complexe.stories.tsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-console */
+import React, { MouseEventHandler, ReactElement } from "react";
+import { ComponentMeta } from "@storybook/react";
+import styled from "styled-components";
+import { Card, Icon } from "../../src";
+import { IconDirection } from "../../src/components/Icon/Direction";
+import { CardStatsDetails } from "../../src/components/Card/CardStats";
+
+export default {
+  title: "Components/Card/Complexe",
+  component: Card,
+} as ComponentMeta<typeof Card>;
+
+interface ExtendedCardProps {
```

Comment:
This sounds suspicious, are you trying to extend the `CardProps` interface?

Like this https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces 

```suggestion
interface SomeNewName extends CardProps  {
```

Reply:
Name is bad, this is specific for the Complex story stuff. Renamed `ComplexCardProps` via commit [f679c2e](https://github.com/Major-Eth/yearn-web-lib/pull/27/commits/f679c2e179e0360f3547d37d6cea3621f3b049da)

Reply:
Looks much better, thanks! 🙇